kura/org.eclipse.kura.core.inventory/src/main/java/org/eclipse/kura/core/inventory/InventoryHandlerV1.java

```diff
@@ -374,4 +355,39 @@ protected String marshal(Object object) {
         }
         return result;
     }
+
+    private String bundleStateToString(int state) {
+        String stateString;
+
+        switch (state) {
+        case Bundle.UNINSTALLED:
```

Comment:
maybe use name() or toString()?

Reply:
@MMaiero It seems that these values are constants. It's not a enum class...